### ISSUE TITLE
Add visual notifications for events

### ISSUE THREAD

**Issue by RetteraDev** [SquonSerq/replay-recorder#3]
We need to create new frame Notification. That frame will take three args( title, description, color ). It should be shown instead of print functions. This frame should have timeout to self destroy

**Comment by Lukumiya** [comment]
Maybe we should use System's notifications?

**Comment by RetteraDev** [comment]
@Lukumiya, yes, we can, but it will be less customized. We can raise win32 windows for warnings, but with our custom frame it'll looks better
We'll make design for all interfaces after made main functional
